.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,90 @@
+name: Bug report
+description: Help us improve InstantSearch
+labels:
+  - triage
+body:
+- type: textarea
+  id: current-behavior
+  attributes:
+    label: 🐛 Current behavior
+    description: A clear and concise description of what the bug is.
+  validations:
+    required: true
+- type: textarea
+  id: reproduction-steps
+  attributes:
+    label: 🔍 Steps to reproduce
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. Go to ...
+      2. Click on ...
+      3. Scroll down to ...
+      4. See error
+
+      You can attach screenshots by dragging and dropping, or selecting and pasting them in the text field.
+  validations:
+    required: true
+- type: input
+  id: reproduction-links
+  attributes:
+    label: Live reproduction
+    description: |
+      A live example helps a lot! Fork the sandbox relevant to the library you're using, reproduce the bug, and paste the URL here.
+
+      <table>
+        <tr>
+          <th>Sandboxes</th>
+          <td><a href="https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/instantsearch.js" target="_blank">InstantSearch.js</a></td>
+          <td><a href="https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/react-instantsearch" target="_blank">React InstantSearch</a></td>
+          <td><a href="https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/react-instantsearch-hooks" target="_blank">React InstantSearch Hooks</a></td>
+          <td><a href="https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/vue-instantsearch" target="_blank">Vue InstantSearch</a></td>
+          <td><a href="https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/angular-instantsearch" target="_blank">Angular InstantSearch</a></td>
+        </tr>
+      </table>
+
+    placeholder: https://codesandbox.io/...
+  validations:
+    required: true
+- type: textarea
+  id: expected-behavior
+  attributes:
+    label: 💭 Expected behavior
+    description: A clear and concise description of what you expected to happen.
+  validations:
+    required: true
+- type: input
+  id: version
+  attributes:
+    label: Package version
+    placeholder: 4.49.2
+  validations:
+    required: true
+- type: dropdown
+  id: os
+  attributes:
+    label: Operating System
+    multiple: true
+    options:
+      - Android
+      - iOS
+      - Linux
+      - macOS
+      - Windows
+- type: dropdown
+  id: browser
+  attributes:
+    label: Browser
+    multiple: true
+    options:
+      - Chrome
+      - Firefox
+      - Internet Explorer
+      - Safari
+      - Other (specify in Current behavior)
```

Comment:
Shouldn't we just provide separate inputs so they can also share version?

Reply:
I'm cautious of having too many fields in this form as well as not sure how to format it when contributors select multiple os or browsers when relevant.

Would a description asking them to add these details in "Current behavior" work?

Reply:
This is what I had in mind, that seems manageable, wdyt?

<img width="466" alt="Capture d’écran 2022-12-19 à 18 19 09" src="https://user-images.githubusercontent.com/5370675/208483115-31bc83e9-2aee-4896-8730-b219171aae60.png">